docs/xks/developer-guide/secrets-management.md

```diff
@@ -63,6 +63,39 @@ spec:
               secretProviderClass: foo
 ```
 
+Additionally, you may want to have your secrets available as environment variables. Extending the example above, the secret `bar` is now also available as the environment variable `BAR`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: foo
+spec:
+  template:
+    spec:
+      containers:
+        - name: foo
+          volumeMounts:
+            - name: secret-store
+              mountPath: "/mnt/secrets-store"
+              readOnly: true
+          env:
+            - name: BAR
+              valueFrom:
+                secretKeyRef:
+                  name: bar
+                  key: bar
+      volumes:
+        - name: secret-store
+          csi:
+            driver: secrets-store.csi.k8s.io
+            readOnly: true
+            volumeAttributes:
+              secretProviderClass: foo
+```
+
+Note that even if you only want to access your secret via an environment variable, you still need to mount the secret store as a volume.
```

Comment:
I am not 100% sure if this is true. I think this is what @NissesSenap told me, but someone should verify this for me.

Reply:
Yes it's true.